Tests/ResultTests/NeverTests.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import XCTest
+import Result
+
+#if swift(>=5.0)
+final class NeverTests: XCTestCase {
+	func testEquatable() {
+		let foo = Result<Int, Never>(1)
+		let bar = Result<Int, Never>(1)
+		XCTAssertTrue(foo == bar)
+	}
+}
+#endif
```

Comment:
This test is unnecessary. If `swift(>=5.0)`, then all of these types are from the standard library. We don't need to test the contents of the standard library.

Could you remove this test?

Reply:
Done